modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -721,7 +747,17 @@
                         "083"
                       ],
                       "pageSize": "3",
-                      "pageIndex": "1"
+                      "pageIndex": "1",
+                      "filter": {
+                        "status": [
+                          "NEW",
+                          "ACCEPTED",
+                          "DECLINED"
+                        ],
+                        "state": "OR",
+                        "city": "Portland",
+                        "country": "USA"
```

Comment:
we probably want to tag some of these to BRD eventually? is there a need to validate these @klwatanabe ? specifically state and country? perhaps evaluate and do this in another ticket?